.github/workflows/ci.yml

```diff
@@ -94,10 +94,10 @@ jobs:
         run: pnpm install
 
       - name: Build types
-        run: pnpm run build:types
+        run: pnpm run build
```

Comment:
**thought (non-blocking):** currently the process is setup in a way that types are build independent of the build process so they can be used for esm and cjs at the same time  
the warning you are currently chasing is just a warning and can be ignored  
it is only important if there are specific cases, but that's not the case for node-pg-migrate, is it? if there is such thing, please point that out to me

Reply:
Ok, I'll remove the last commit and mark related comments as resolved 🤝